docs/readmes/orc8r/upgrade_1_4.md

```diff
@@ -78,11 +78,23 @@ terraform refresh
 
 ## Terraform apply
 
+Terraform the upgrade. We'll also need to manually rescale the Prometheus
+deployment, as a workaround for quirks in how its deployment handles upgrading.
+
 Apply the upgrade
 
 ```bash
 terraform apply     # Check this output VERY carefully!
 ```
 
-After applying Terraform changes, all your application components should be
-upgraded.
+Once the above command indicates it's applying the Helm upgrade, switch to
+a separate terminal tab and manually scale the Prometheus deployment. You'll
+have a 10-minute window to run the command. You can re-run the Terraform
```

Comment:
What's the point of saying there's a 10 minute window? Seems kinda stressful lol. I'd also mention briefly the reasoning for running this command, and also say that data loss will be minimal if not none. e.g. "Rescaling the prometheus deployment allows for the upgraded pod to obtain the lock on the data directory. Only a few datapoints will be lost during this process, if any at all, due to how quickly this process happens."